src/website/app/demos/routes.js

```diff
@@ -17,58 +17,86 @@
 /* @flow */
 export default {
   button: {
+    description:
+      'A Button should be used whenever you need to trigger an action in your app. The color of the button should be chosen according to the intent of the action.',
```

Comment:
Buttons are used to trigger actions or state changes in your app.

Reply:
same as above. what about something like "Use Buttons to trigger actions"...?

Reply:
I read the Button description, then the Card one and then CardBlock. It seemed like Button was the outlier